layouts/default/numpad_6x5/default_numpad_6x5/keymap.c

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2021 NachoxMacho   
+*   
+* This program is free software: you can redistribute it and/or modify   
+* it under the terms of the GNU General Public License as published by   
+* the Free Software Foundation, either version 2 of the License, or   
+* (at your option) any later version.   
+*   
+* This program is distributed in the hope that it will be useful,   
+* but WITHOUT ANY WARRANTY; without even the implied warranty of   
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the   
+* GNU General Public License for more details.   
+*   
+* You should have received a copy of the GNU General Public License   
+* along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.   
```

Comment:
```suggestion
/* Copyright 2021 NachoxMacho
*
* This program is free software: you can redistribute it and/or modify
* it under the terms of the GNU General Public License as published by
* the Free Software Foundation, either version 2 of the License, or
* (at your option) any later version.
*
* This program is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
* GNU General Public License for more details.
*
* You should have received a copy of the GNU General Public License
* along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.
```